samples/client/js/package.json

```diff
@@ -10,7 +10,7 @@
     "load-translation": "load-translation --directory `pwd`/src/sources --product CoreSDKClient --component ui --host http://localhost:8091 --version 1.0 --languages zh,en"
   },
   "dependencies": {
-    "@vip/vip-core-sdk": "file:../../vip-vip-core-sdk-0.1.0.tgz",
+    "@vip/vip-core-sdk": "file:../../../vip-vip-core-sdk-0.1.0.tgz",
```

Comment:
Why introduce this change here?

Reply:
Actually it is fixed here: https://github.com/vmware/singleton/pull/186